hbase-client/src/main/java/org/apache/hadoop/hbase/client/AsyncAdminBuilder.java

```diff
@@ -50,21 +50,21 @@
    * Set the base pause time for retrying. We use an exponential policy to generate sleep time when
    * retrying.
    * @return this for invocation chaining
-   * @see #setRetryPauseForCQTBE(long, TimeUnit)
+   * @see #setRetryPauseForServerOverloaded(long, TimeUnit)
    */
   AsyncAdminBuilder setRetryPause(long timeout, TimeUnit unit);
 
   /**
-   * Set the base pause time for retrying when we hit {@code CallQueueTooBigException}. We use an
+   * Set the base pause time for retrying when we hit {@code ServerOverloadedException}. We use an
    * exponential policy to generate sleep time when retrying.
    * <p/>
    * This value should be greater than the normal pause value which could be set with the above
-   * {@link #setRetryPause(long, TimeUnit)} method, as usually {@code CallQueueTooBigException}
+   * {@link #setRetryPause(long, TimeUnit)} method, as usually {@code ServerOverloadedException}
    * means the server is overloaded. We just use the normal pause value for
-   * {@code CallQueueTooBigException} if here you specify a smaller value.
+   * {@code ServerOverloadedException} if here you specify a smaller value.
    * @see #setRetryPause(long, TimeUnit)
    */
-  AsyncAdminBuilder setRetryPauseForCQTBE(long pause, TimeUnit unit);
```

Comment:
Ah, this is a silly mistake on my part. I was thinking I didnt need that since master/hbase3 is unreleased, but the same method does exist in branch-2 so I will need it here and there.  Will fix.